build.assets/charts/Dockerfile

```diff
@@ -1,3 +1,8 @@
+# DEPRECATED: Images from this dockerfile are not published for v15 and above
+# https://goteleport.com/docs/changelog/#heavy-container-images-are-discontinued
+# Teleport images are built from Dockerfile-distroless
+# TODO(hugoShaka): cleanup the Makefile docker/image targets and remove this file.
+
```

Comment:
I can't remove thisd today because there are Makefile targets using it. We must clean the Makefile first and it will take some time.